venonactl/pkg/templates/kubernetes/daemonset.dind-lv-monitor.vp.yaml

```diff
@@ -34,6 +34,27 @@ spec:
       containers:
         - image: {{ if ne .DockerRegistry ""}} {{- .DockerRegistry }}/codefresh/dind-volume-utils:v5 {{- else }}codefresh/dind-volume-utils:v5{{- end}}
           name: lv-cleaner
+      {{if or  .Storage.LocalVolumeMonitor.Limits .Storage.LocalVolumeMonitor.Requests }}
```

Comment:
R U sure this check is still working? because we are always initializing this values so they are never nil